specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -492,6 +493,17 @@ New versions are released by changing the version in
 New releases are published under the image name
 [registry.hub.docker.com/versatiledatakit/vdk-cicd-base-gui](https://hub.docker.com/r/versatiledatakit/vdk-cicd-base-gui)
 
+##### Quickstart vdk image
+
+The VDK Operations UI ships with `quickstart-vdk`. The CI/CD pipeline builds the
+image on every change to `data-pipelines` and `shared`. The image is uploaded to
+dockerhub and can then be used in the helm charts for `quickstart-vdk`.
+
+The image contains:
+
+1. An nginx server, configured to listen on port 8091
+2. The Angular UI application running on the server
+
```

Comment:
May describe the diff between the E2e test and Quickstart images?
The FROM instruction is obvious, and a person may track whatever the base is (hopefully). The suggestion here is, to explain the requirements towards the UI distro build, before effectively copying it onto those both images? 
The reader may wonder why is another image needed? Any details about image specifics, should be present in both images manifests, so their purpose may be defined.